### PR TITLE
chore: bump deps and readme

### PR DESCRIPTION
**Pull request by greyscaled**
This is a simple PR that:

- bumps the bundler-managed dependencies, including those dependencies for MacOS arm64
- greatly simplifies the README, dropping unnecessary stuff

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ title: GlareDB Docs
 description: Documentation for GlareDB
 theme: just-the-docs
 
-url: https://docs.glaredb.com
+url: https://glaredb.com
```
Comment (talagluck) [inline _config.yml:5]: Why did we change this URL?

File: README.md [hunk 2]
```diff
@@ -16,118 +16,40 @@ on contributing, see our [Contributing guidelines].
...
-
-# Attempts to automatically fix any lint violations
-npm run lint:fix
-```
-
-#### Spell checking
-
-See `cspell.json` for spell checking configuration. The `patterns` are regular
-expressions for which spell checking is ignored (example: in code blocks).
-The `words` array is a list of words to ignore.
```
Comment (talagluck) [inline README.md:89]: I found the linting stuff pretty helpful when working on the re-arch. I know we also have it for the marketing site, but it was nice to have it on hand here. Do you feel strongly about removing it?

**Review by talagluck — APPROVED** [review]
Looks good - thanks for the update! A couple of questions on the decision to remove stuff.

File: README.md (hunk 2 above)
Comment (greyscaled) [reply README.md:89]: I don't feel too strongly, no

File: _config.yml (hunk 1 above)
Comment (greyscaled) [inline _config.yml:5]:
```suggestion
url: https://docs.glaredb.com
```

File: README.md (hunk 2 above)
Comment (greyscaled) [reply README.md]: I re-added a simplified version of it!

**Pull request merged**